schema/manager.go

```diff
@@ -43,9 +40,13 @@ type Manager struct {
 	policies    []*Policy
 	Extensions  []*Extension
 	namespaces  map[string]*Namespace
+	mu          sync.RWMutex
```

Comment:
Exactly. Gohan uses "mutex", that is why I mentioned local convention (which should be favored IMO). git grep -E "sync\..*Mutex" shows mutex, mu, m, mtx and lock. But anyway, this is a minor issue.